cmd/osbuild-worker/main.go

```diff
@@ -189,6 +198,39 @@ var run = func() {
 		logrus.Fatalf("Could not load config file '%s': %v", configFile, err)
 	}
 
+	logrus.SetReportCaller(true)
+	logrus.AddHook(&common.BuildHook{})
+	logrus.SetOutput(os.Stdout)
+	logLevel, err := logrus.ParseLevel(config.Logging.Level)
+	if err == nil {
```

Comment:
(nitpick/drive-by) maybe switch this around to `if err != nil {` first as this the most common way in go to deal with errors? I know it's minor but when I read code that does not follow this pattern I immediately stop in my flow as this signal "something here is special" (but here it seems it is not (but feel free to ignore of course)

Reply:
Amended.